src/extension.ts

```diff
@@ -251,21 +252,29 @@ export function activate(context: vscode.ExtensionContext) {
         }
     }
 
-    function saveProject() {
-        // Display a message box to the user
-        // let wpath = vscode.workspace.rootPath;
-        const workspace0 = vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0] : undefined;
-        let wpath = workspace0 ? workspace0.uri.fsPath : undefined;
+    function saveProject(node: any = undefined) {
```

Comment:
TSLint is complaining about this syntax: `no-unnecessary-initializer`. 
Use `node?: any` instead